ci/bin/format.dart

```diff
@@ -334,7 +328,7 @@ class ClangFormatChecker extends FormatChecker {
     );
   }
 
-  /*late*/ File clangFormat;
+  late File clangFormat;
```

Comment:
This looks like it could be `late final`, or maybe even `static final` since it's always the same thing and statics are lazy in Dart.